### PR TITLE
Add support for log colours

### PR DESCRIPTION
**Pull request by catornot**
another pr woah, it's a draft since I would like to make a rrplug release

related to https://github.com/R2Northstar/NorthstarLauncher/pull/823

~~any color suggestions?~~

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
What's needed to make the rrplug release?

File: src/presence.rs [hunk 1]
```diff
@@ -35,6 +35,8 @@ pub fn run_presence_updates(sqvm: NonNull<HSquirrelVM>) {
     };
 }
 
+// "Localize_001" is a thing
+
```
Comment (GeckoEidechse) [inline src/presence.rs:39]: Leftover comment or added annotation on purpose?

Comment (catornot) [reply src/presence.rs:39]: for future stuff

**Comment by catornot** [comment]
> What's needed to make the rrplug release?

`¯\_(ツ)_/¯`

probably I will just have to check if docs make sense

File: Cargo.toml [hunk 2]
```diff
@@ -6,7 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rrplug = "4.0.0"
+# rrplug = "4.0.0"
+rrplug = "=4.1.0"
```
Comment (GeckoEidechse) [inline Cargo.toml:9]: out of curiosity, is there a reason you are locking to that specific version in `Cargo.toml` as well? ^^

**Review by GeckoEidechse — APPROVED** [review]
Confirmed working in testing together with https://github.com/R2Northstar/NorthstarLauncher/pull/823 :D

image

File: Cargo.toml (hunk 2 above)
Comment (catornot) [reply Cargo.toml:9]: because I might not follow semver for rrplug :trollface:

Comment (GeckoEidechse) [reply Cargo.toml:9]: :dread:

**Pull request merged**